tests/test_upload.py

```diff
@@ -285,3 +287,20 @@ def none_upload(*args, **settings_kwargs):
     assert "pypipassword" == upload_settings.password
     assert "pypiuser" == upload_settings.username
     assert "/foo/bar.crt" == upload_settings.cacert
+
+
+def test_check_status_code(make_settings, capsys):
```

Comment:
I think the test name could be more descriptive. Something like:

```suggestion
def test_check_status_code_for_wrong_url(make_settings, capsys):
```

Reply:
Fixed in `ea8d010`.

Reply:
Fun GitHub facts:

- Commit hashes will be linked if you leave them bare, ala ea8d010
- Clicking one of the buttons in a ["Suggested change" box](https://help.github.com/en/articles/incorporating-feedback-in-your-pull-request) will save you some typing. :)

Speaking of commits: I think it's a good practice to use a short one-line title (max 50 characters), and a longer description. This will make them easier to read in a number of places. Also, I think a long title can indicate that you're doing too much in one commit. :)

Here's one of many articles on the subject: https://chris.beams.io/posts/git-commit/. Admittedly, I don't strictly adhere to this (often leaving out a description), but we can aspire.

Reply:
Good points, thank you! I'll keep them in mind for my commits. :)